doc/source/quickstart.rst

```diff
@@ -14,30 +14,27 @@ In the Python interpreter, you may then call::
 which should run without error if the package has been correctly installed.
 
 
-Apply 2D scattering to a 32x32 random image
+Apply 2D scattering to a 32×32 random image
 -------------------------------------------
 
-The following code imports ``torch`` and the ``Scattering2D`` class, which
-implements the 2D scattering transform. It then creates an instance of this
-class to compute the scattering transform at scale ``J = 2`` of a 32x32 image
+The following code imports ``numpy`` and the ``Scattering2D`` class, which implements the 2D scattering transform. It
+then creates an instance of this class to compute the scattering transform at scale ``J = 2`` of a 32×32 image
 consisting of Gaussian white noise::
 
-    import torch
-    from kymatio import Scattering2D
+    import numpy as np
+    from kymatio.numpy import Scattering2D
     scattering = Scattering2D(J=2, shape=(32, 32))
-    x = torch.randn(1, 1, 32, 32)
+    x = np.random.rand(1, 1, 32, 32)
     Sx = scattering(x)
-    print(Sx.size())
+    print(Sx.shape)
 
 This should output::
 
-    torch.Size([1, 1, 81, 8, 8])
+    (1, 1, 81, 8, 8)
 
-This corresponds to 81 scattering coefficients, each corresponding to an
-8x8 image.
+This corresponds to 81 scattering coefficients, each corresponding to an 8×8 image.
 
 Check out the :ref:`user-guide` for more scattering transform examples.
 
-The performance of the scattering transform depends on the specific backend
-that's in use. For more information on switching backends to improve
-performance, see the :ref:`backend-story` section.
+The performance of the scattering transform depends on the frontend as well as the specific backend that's in use. For
```

Comment:
avoid apostrophe

Reply:
I don't know. Do we need to be that formal?